roles/inactivity-shutdown/files/inactivity-shutdown

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
-if [[ ! $(who) ]]; then
+if [[ ! $(who) && ! $(netstat -natp | grep -e 3389.*ESTABLISHED) ]]; then
```

Comment:
Netstat ist deprecated:
https://dougvitale.wordpress.com/2011/12/21/deprecated-linux-networking-commands-and-their-replacements/

Reply:
Mir aber egal, falls du das nicht umbauen willst, bitte einfach mergen. Danke für deine Arbeit